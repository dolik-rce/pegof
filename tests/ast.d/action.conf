input ast.d/action.peg
ast
