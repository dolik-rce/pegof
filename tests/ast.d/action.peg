X <- "X" { printf("action!\n"); }
