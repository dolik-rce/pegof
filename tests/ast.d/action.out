GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          STRING X
        TERM
          ACTION printf(\"action!\\n\");
